tests.py

```diff
@@ -81,13 +90,13 @@ def _test_default_attrs(self, ctor, *args):
         obj = ctor(*args)
         # Check defaults
         self.assertEqual(obj.dialect.delimiter, ',')
-        self.assertEqual(obj.dialect.doublequote, True)
+        self.assertIs(obj.dialect.doublequote, True)
```

Comment:
`assertIs` does not seem to be working on 3.6. I will get back to  `assertEqual`